wercker.yml

```diff
@@ -6,6 +6,17 @@ build:
             name: initialize git submodules
```

Comment:
Random comment for Erick https://github.com/matsengrp/linearham/issues/39#issuecomment-273210340